<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainForm</class>
 <widget class="QWidget" name="CMainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mchat</string>
  </property>
  <widget class="QListWidget" name="listFriend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>221</width>
     <height>341</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="gridSize">
    <size>
     <width>0</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="btnHead">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNick">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnAddFriend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>添加好友</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnCover">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
